### PR TITLE
Add docs to explain cursor drawing behavior.

### PR DESCRIPTION
**Pull request by GaryQian**

### CODE REVIEW

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 1]
```diff
@@ -417,6 +417,11 @@ class EditableText extends StatefulWidget {
   /// How thick the cursor will be.
   ///
   /// Defaults to 2.0
+  ///
+  /// The cursor will draw under the text. The cursor width will extend
```
Comment (xster) [inline packages/flutter/lib/src/widgets/editable_text.dart]: Out of scope for this PR but we should probably draw on top of the text

**Review by xster — COMMENTED** [review]
LGTM

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 2]
```diff
@@ -417,6 +417,11 @@ class EditableText extends StatefulWidget {
...
+  /// to the right of the boundary between characters for right-to-left text
```
[the remark this answers is not in the document]
Comment (GaryQian) [reply packages/flutter/lib/src/widgets/editable_text.dart]: No it is the other way around. The cursor extends to the RIGHT, which means downstream for rtl text.

[the remark this answers is not in the document]
Comment (GaryQian) [reply packages/flutter/lib/src/widgets/editable_text.dart]: Oh wait, my bad mixed up rtl with ltr
You were right.

**Pull request merged**